waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -620,7 +626,8 @@
                                :labels {:app "test-app-9999"
                                         :waiter/cluster "waiter"
                                         :waiter/service-hash "test-app-9999"}
-                               :annotations {:waiter/service-id "test-app-9999"}
+                               :annotations {:waiter/revision-timestamp "v20200922T202222Z"
```

Comment:
Should we switch all of these strings to ISO 8601 format? You should be able to do this automatically with `sed`:
```
$ sed -E 's/v([0-9]{4})([0-9]{2})([0-9]{2})T([0-9]{2})([0-9]{2})([0-9]{2})Z/\1-\2-\3T\4:\5:\6Z/g' <<< v20200922T202222Z
2020-09-22T20:22:22Z
```

Reply:
Changed to ISO format.